src/ceph-daemon/ceph-daemon

```diff
@@ -1342,7 +1342,7 @@ def command_ceph_volume():
     if args.fsid:
         make_log_dir(args.fsid)
 
-    (uid, gid) = extract_uid_gid()
+    (uid, gid) = (os.geteuid(), os.getegid())
```

Comment:
actually, root:root.  c-v runs as root in the container.  as long as these files are readable by root all is well.

Reply:
updated to use uid/gid 0 (root:root)